docs/onboarding/rpc/chain.md

```diff
@@ -117,24 +117,28 @@ Follows the chain from a given sequence and streams blocks from chain connects a
     timestamp: number
     work: string
     main: boolean
-    transactions: Array<{
+    transactions:{
```

Comment:
I'd like to stick with `Array<>` for these -- I think leading with `Array` makes the nested array + inline object types easier to understand instead of trailing `[]`.

Reply:
Addressed